examples/cloudml-bee-health-detection/trainer/model.py

```diff
@@ -65,19 +65,16 @@ def create_classifier(config, parameters):
     Returns:
         `tf.estimator.DNNClassifier` with specified features and architecture.
     """
-    # Mean and Standard Deviation Constants for normalization.
-    depth_mean = np.float32(parameters.depth_mean)
-    depth_std = np.float32(parameters.depth_std)
-
     # Columns to be used as features.
-
-    depth = tf.feature_column.numeric_column(
-        'depth',
-        normalizer_fn=(lambda x: (x - depth_mean) / depth_std))
+    subspecies = tf.feature_column.categorical_column_with_vocabulary_list(
+        'subspecies',
+        ['Carniolan','Italian','Russian','Other'])
```

Comment:
Don't forget the spaces between list elements.

Reply:
will fix